routing/localchans/manager.go

```diff
@@ -76,6 +76,10 @@ func (r *Manager) UpdatePolicy(newSchema routing.ChannelPolicy,
 			return nil
 		}
 
+		// Mark this channel as found by removing it. chansToUpdate
+		// will be used to report absent channels later on.
+		delete(chansToUpdate, info.ChannelPoint)
+
```

Comment:
maybe move this to below the `r.updateEdge()` line so that we can also report back to the user if a channel update could not happen at all (even if the channel was found)